### DISCUSSION TITLE
Missing module - breaks Zoe Depth Map preprocessor

### DISCUSSION THREAD

**Discussion by JeffJag-ETH** [comfyanonymous/ComfyUI#2589] [General]
ModuleNotFoundError: No module named 'midas.dpt_depth'

that's part is seen when running the .bat to start comfy

so then when trying to use a Zoe Depth Map preprocessor from the comfyui controlnet aux library/install it gives a similar error. https://github.com/Fannovel16/comfyui_controlnet_aux/tree/eb6eb79e0e72b98235dba7397c8165eb4077116c install
![image](https://github.com/comfyanonymous/ComfyUI/assets/119813682/5a1ebc30-2734-42c2-a586-96e06eebf0e4)

I feel like it's super taboo to put in a bug report when there's nothing else in that entire category, so I hope I didn't offend. 
Actually, I just don't think Comfy has any bugs at all. that's why. makes sense.

Any help appreciated. I don't like the midas processor as much as zoe

**Comment by SashaRoberts22** [comment]
has anyone found the answer to this?
